pass-authz-acl/src/main/java/org/dataconservancy/pass/authz/acl/PolicyEngine.java

```diff
@@ -70,14 +72,12 @@ public void updateSubmission(URI uri) {
         final Set<URI> authReaders = new HashSet<>();
         final Set<URI> authWriters = new HashSet<>();
 
-        if (submission.getUser() != null) {
-            authReaders.add(submission.getUser());
+        // If a submission is "submitted=true", then it's frozen.
+        if (submission.getSubmitted() == null || !submission.getSubmitted()) {
 
```

Comment:
I'm not familiar with when this will run - would it be on every submission update? so presumably when an update happens in which `Submission.submitted` becomes true, the authWriters will get removed at that point

Reply:
Yes, exactly.  every submission update.  The other ticket #40 is to bring the logic into alignment with the re-imagined submission states for the proxy use case, with respect to read-only and read-write states,